main.yml

```diff
@@ -23,7 +23,7 @@
       - name: gather export (with password authentication)
         # shell with delegate_to: localhost works better that ansible.builtin.raw  
         ansible.builtin.shell: >-
-          sshpass -p '{{ ansible_ssh_pass }}' ssh -o StrictHostKeyChecking=no {{ ansible_user }}@{{ inventory_hostname }} /export
+          sshpass -p '{{ ansible_ssh_pass }}' ssh -o StrictHostKeyChecking=no {{ ansible_user }}@{{ inventory_hostname }} -p {{ ansible_port }} /export
```

Comment:
I suggest to do in one of the ways:
- use `{{ ansible_port | default('22') }}` everywhere where you added `{{ ansible port }}`
- add `set_facts` task with something like `ansible_port = {{ ansible_port | default("22") }}`

The reason for that is backward compatibility with already created inventory without `ansible_port`

Reply:
Thanks for the suggestion:)